homeassistant/components/climate/oem.py

```diff
@@ -0,0 +1,139 @@
+"""
+OpenEnergyMonitor Thermostat Support.
+
+This provides a climate component for the ESP8266 based thermostat sold by
+OpenEnergyMonitor.
```

Comment:
Be sure to link to the documentation page as well. (Link won't be valid yet.) Use the standard boilerplate. Example here: https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/climate/nest.py#L4